resources/js/crud.js

```diff
@@ -252,6 +252,28 @@ function registerEntityCalendarModal() {
         entityCalendarSubForm.fadeIn();
         loadCalendarDates(entityCalendarField.val());
     }
+
+    $('.entity-calendar-subform input[name="length"]').focusout(function () {
+        let url = $(this).data('url').replace('/0/', '/' + entityCalendarField.val() + '/')
```

Comment:
Can we avoid hitting the api when the field is empty to preserve server resources? :)